README.md

```diff
@@ -1,58 +1,71 @@
-# unified-codes
+# Universal Codes
 
-## Getting started
+Welcome! This is Universal Codes!
 
-1. Install dependencies: `npm install`.
-2. Follow application instructions below.
+This is a responsive web application over a GraphQL API.
 
-## Applications
+It is built using Typescript, with a React Frontend and an Apollo Server with a DGraph database. The frontend code is heavily 'borrowed' from the [Notify frontend](https://github.com/openmsupply/notify/tree/main/frontend)
 
-### data-loader
+The test framework is jest; functional areas are separated into packages and managed with [lerna](https://lerna.js.org/)
 
-Application for parsing product data and populating Dgraph database.
+## Usage
 
-1. Setup Dgraph (see instructions [here](unified-codes/tools/scripts/dgraph/README.md)).
-3. Populate graph schema and demo data: `npx nx serve data-loader`.
-4. Dgraph is now running, you can access at [http://localhost:9080](http://localhost:9080).
+- Install dependencies (Using node v20+ and yarn):
+  - `yarn install`
+- Setup Dgraph (see instructions [here](unified-codes/tools/scripts/dgraph/README.md))
+  - You can then use the [explorer](https://play.dgraph.io/?latest) (server URL is http://localhost:8080)
+- Load data into DGraph:
+  - `yarn load-data`
```

Comment:
Any reason we shouldn't combine `load-data` and `update-dgraph-schema`.
I guess it's good having both options...

Reply:
It's a good question... I think having the separate update-dgraph-schema is useful, if we don't want to a full reload of the data, but we should always run it after the load... will update script now 🚀

Reply:
[82b0f35](https://github.com/msupply-foundation/unified-codes/pull/430/commits/82b0f35feb23dee42ca027d1404f8858ba254052) ... we'll need to come back to this whole data-loader piece... its probably quite useful from a development standpoint, but we'll phase it out for production, and we'll want to review how we manage the schema then too

Reply:
Makes a lot of sense.